cpp/example_code/general/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+﻿#Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#SPDX-License-Identifier: Apache-2.0
+# snippet-start:[general.cpp.starter.cmakelists]
+cmake_minimum_required(VERSION 3.3)
+# For the AWS C++ SDK build use shared libraries.

```

Comment:
I suggest you add a blank line before each comment for readability. I'm not going to waste your time by repeating this, but you should add blanks whever necessary to make the files easier for others.